### ISSUE TITLE
License question

### ISSUE THREAD

**Issue by coderbot16** [World-Control-Team/World-Control#23]
I was evaluating this mod for inclusion in my pack, and noticed that it doesn't explicitly state a license anywhere, meaning it is All Rights Reserved by default. The way I understand it, this effectively means that any usage of the mod outside of the guarantees offered by GitHub and Curse is illegal (such as a private non-curse pack for a server). My question is whether this state of affairs is by design, or whether a license simply hasn't been chosen yet. In the case of the latter, a good choice would be a copyleft license such as the LGPL or GPL, or even one of the CC licenses.

**Comment by xbony2** [comment]
@coderbot16 Basically a lot of the code/assets is derived from the original Nuclear Control and I don't know if it's enough to for this be considered an original project that I can give a license. The original Nuclear Control license is [here](https://github.com/xbony2/Nuclear-Control/blob/master/LICENSE). Anyway, you're absolutely free to use in a modpack. The original creator is a Russian no longer involved with Minecraft or Nuclear Control, while I'm a busy student, so no one's going to sue or anything.

**Comment by coderbot16** [comment]
Thanks for the info, good to hear!

With regards to the licensing, the way I understand it is if the code/assets in this project are derived from the original Nuclear Control, licensed under the BSD 3-clause license, then the BSD 3-clause license applied to it would carry over to World Control on account of the following clause:

`Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.`

So if the code or assets in World Control are derivative works of Nuclear Control, not only does that mean that you should give it a license, it would mean that World Control would need to be licensed under the BSD-3 clause as well.

**Comment by mincrmatt12** [comment]
No, the BSD 3 clause isn't viral, the clause you reference refers to the fact that non derivative works need to include the license and warranty disclaimer.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by coderbot16** [comment]
That is correct. However, wouldn't the BSD-3 license would still require some sort of acknowledgement that the project is a fork/derivative work of BSD-3, even if the project as a whole uses a different license?
